backbone.js

```diff
@@ -1425,9 +1425,9 @@
   var methodMap = {
     'create': 'POST',
     'update': 'PUT',
-    'patch':  'PATCH',
```

Comment:
We would have to either disable the rule, or set it to only warn (and then tolerate the fact that it's always going to throw warnings).

I'm fine either way. If we want to do either of the above, just let me know and I'll open a PR which reverts only the cases where the values were aligned.